Lux/lux_files/kaleido.json

```diff
@@ -197,6 +204,169 @@
             "a": { "functions": [ "funky_menu" ] },
             "b": 5
         },
+        {
+            "name": "start_audio",
+            "type": "audio_additive_fn",
+            "channel": "high",
+            "sensitivity": 200.0,
+            "offset": 0.0
+        },
+        {
+            "name": "expand_audio",
+            "type": "audio_additive_fn",
+            "channel": "bass",
+            "sensitivity": 300.0,
+            "offset": 0.0
+        },
+        {
+            "name": "spin_audio",
+            "type": "audio_additive_fn",
+            "channel": "mid",
+            "sensitivity": 400.0,
+            "offset": 0.0
+        },
+        {
+            "name": "phase_audio",
+            "type": "audio_additive_fn",
+            "channel": "volume",
+            "sensitivity": 250.0,
+            "offset": 0.0
+        },
+        {
+            "name": "swirl_audio",
+            "type": "audio_additive_fn",
+            "channel": "bass",
+            "sensitivity": 350.0,
+            "offset": 0.0
+        },
+        {
+            "name": "alternate_audio",
+            "type": "audio_additive_fn",
+            "channel": "volume",
+            "sensitivity": 180.0,
+            "offset": 0.0
+        },
+        {
+            "name": "level_start_audio",
+            "type": "audio_additive_fn",
+            "channel": "mid",
+            "sensitivity": 220.0,
+            "offset": 0.0
+        },
+        {
+            "name": "rings_audio",
+            "type": "audio_additive_fn",
+            "channel": "high",
+            "sensitivity": 150.0,
+            "offset": 0.0
+        },
+        {
+            "name": "freq_audio",
+            "type": "audio_additive_fn",
+            "channel": "high",
+            "sensitivity": 120.0,
+            "offset": 0.0
+        },
+        {
+            "name": "amp_audio",
+            "type": "audio_additive_fn",
+            "channel": "bass",
+            "sensitivity": 280.0,
+            "offset": 0.0
+        },
+        {
+            "name": "birthday_bass_boost",
+            "type": "audio_multiplicative_fn",
+            "channel": "bass",
+            "base_multiplier": 2.0,
+            "sensitivity": 5.0
+        },
+        {
+            "name": "celebration_sparkle",
+            "type": "audio_modulate_fn",
+            "channel": "high",
+            "depth": 100.0,
+            "frequency": 8.0
+        },
+        {
+            "name": "party_energy",
+            "type": "audio_additive_fn",
+            "channel": "volume",
+            "sensitivity": 500.0,
+            "offset": 0.0
+        },
+        {
+            "name": "start_combined",
+            "type": "adder_float",
+            "a": { "functions": [ "start_slider" ] },
+            "b": { "functions": [ "start_audio" ] }
```

Comment:
Here, the audio function is added to slider and fed into integrator.
To get the desired effect, feed slider into integrator as before and add audio function to integrator. That way the audio pulsations will happen on top of the integrator's effect, which will be easier for user to interpret.